subprojects/docs/src/docs/userguide/running-builds/introduction/dependency_management_basics.adoc

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_basics]]
+= Dependency Management Basics
+
+Gradle has built-in support for *dependency management*.
+
+image::gradle-basic-7.png[]
+
+Dependency management is an automated technique for declaring, resolving, and using functionality required by a project.
+
+Gradle build scripts defines a process to build projects that contain some dependencies.
+Dependencies refer to JARs, plugins, libraries, or source code that support building your project.
+
+The Gradle build system makes it easy to include external binaries or other library modules to your build as dependencies.
+The dependencies can be located on your machine or in a remote repository, and any transitive dependencies they declare are also automatically included.
+
+== Version Catalog
+
+Version catalogs are a way of centralizing your dependency declarations by creating a `libs.versions.toml` file.
+
+The catalog makes sharing dependencies and version configurations between subprojects simple.
+It also allows teams to enforce versions of libraries and plugins in large projects.
+
+The version catalog typically contains four sections:
+
+1. [versions] to declare the version numbers that plugins and libraries will reference.
+2. [libraries] to define the libraries used in the build files.
+3. [bundles] to define a set of dependencies.
+4. [plugins] to define plugins.
+
+[source,gradle/libs.versions.toml]
+----
+[versions]
+testHelperVer = "7.4.1"
+
+[libraries]
+gradle-helper = "com.example.gradle:app-helper:10.1.1"
+gradle-test-helper = { group = "com.example.gradle", name = "app-test-helper", version.ref = "testHelperVer" }
+
+[plugins]
+tutorial-application = { id = "com.tutorial.application" }
+----
+
+The file is located in the `gradle` directory so that it can be used by Gradle and IDEs automatically.
+The version catalog should be checked into subversion: `gradle/libs.versions.toml`.
```

Comment:
```suggestion
The version catalog should be checked into source control: `gradle/libs.versions.toml`.
```

Subversion is just one type of source control. Git is the much more popular alternative. We shouldn't need to specify which source control library to use